### PR TITLE
WAN enhancements

### PR DESCRIPTION
**Pull request by emrahkocaman**
- Config changes for new WANQueueFulBehavior
- New REST method to allow clearing WRQs.

### CODE REVIEW

**Comment by emrahkocaman** [comment]
verify

File: hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java [hunk 1]
```diff
@@ -323,6 +325,25 @@ private void handleWanSyncMap(HttpPostCommand command) throws UnsupportedEncodin
         textCommandService.sendResponse(command);
     }
 
+    private void handleWanClearQueues(HttpPostCommand command) throws UnsupportedEncodingException {
```
Comment (mesutcelik) [inline hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java]: That must be called after WAN Rep Stopped via ManCenter. right?

**Comment by mesutcelik** [comment]
👍 with one minor comment

File: hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java (hunk 1 above)
Comment (emrahkocaman) [inline hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpPostCommandProcessor.java]: Yes but WAN replication is paused/resumed automatically when clear queues command is in progress.

**Pull request merged**